src/Sarif/Visitors/SarifVersionOneToCurrentVisitor.cs

```diff
@@ -189,7 +191,59 @@ public static Invocation CreateInvocation(InvocationVersionOne v1Invocation, Run
             return invocation;
         }
 
-        public static LogicalLocation CreateLogicalLocation(LogicalLocationVersionOne v1LogicalLocation)
+        internal Location CreateLocation(LocationVersionOne v1Location)
```

Comment:
As far as I can see nobody is calling this yet (at least not from this file). Can we wait to add it?

Reply:
Yeah I have been working down the line and then moving code out to a waiting area. Missed this one